api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -89,4 +89,14 @@ public GroupsApi groupsApi(@Qualifier(ALL_OF_US_API_CLIENT) ApiClient apiClient)
     api.setApiClient(apiClient);
     return api;
   }
+
+  @Bean(name = "user")
```

Comment:
public constant

Also, if you want to match what I did for the cluster client:
https://github.com/all-of-us/workbench/blob/a9514f7773598239cbf57d7d909b20f8a5700377/api/src/main/java/org/pmiops/workbench/notebooks/NotebooksConfig.java#L21

Reply:
Done.